src/Coypu/Coypu.csproj

```diff
@@ -1,219 +1,26 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
     <ProductVersion>9.0.30729</ProductVersion>
-    <SchemaVersion>2.0</SchemaVersion>
     <ProjectGuid>{0495F7A7-A1A9-422C-BE9D-6D9F9BD4E97C}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>Coypu</RootNamespace>
-    <AssemblyName>Coypu</AssemblyName>
-    <TargetFrameworkVersion>v3.5</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
-    <FileUpgradeFlags>
-    </FileUpgradeFlags>
-    <OldToolsVersion>3.5</OldToolsVersion>
-    <UpgradeBackupLocation />
-    <TargetFrameworkProfile />
+    <TargetFramework>net471</TargetFramework>

```

Comment:
Thank you for this awesome PR! I just installed it to be able to use NUnit 3.11!
Just one request from my side:
Please do not upgrade to latest .NET version. Otherwise project cannot be referenced by projects using an older .NET framework. You could specify .NET 4.5 here and everything still works fine.

Reply:
I'm aware of that that's why I commented as breaking change. The reason is that in my current company we upgrade regularly .net framework and we currently are using net471 and slowly migrate towards .net standard 2.0

Reply:
Before you merge this, I would highly recommend targeting net461 or lower, not net471. There's no benefit in keeping this project targeting a newer netfx version unless you're specifically using new features. 

**This change would likely prevent almost every single user of this library from upgrading.** Most enterprise environments are not as flexible with netfx versions as yours.

Any additional framework changes in the future should focus on making this library .net standard 2.0 compatible so it can be used with both .net core and .net framework.

Reply:
yes, I'm aware of this. I was thinking to release two versions of coypu 3.14.0.452 (as for .NET 4.5.2) and 3.14.0.471 (as for .NET 4.7.1) but master I will keep it as net471 and in first half of 2019 migrate it to .net standard and ultimately to .net core (but thats end of 2019, probably even 2020)

Reply:
cool, even better. Anyway I've opened discussion under https://github.com/featurist/coypu/issues/192 to be more visible for everyone